docs/features/readonly-variables-and-parameters.md

```diff
@@ -0,0 +1,61 @@
+### High Level Language Proposal:  
+Allow 'readonly' to be used on local variables and on parameters. 
+
```

Comment:
> I think generalizing const could be a nicer approach

That would be fine *if* we then allowed the same for fields.  However, i doubt we'd do that.  And if we are in the state where we have const and readonly fields, then having only const locals (with inconsistent meaning vs fields) then that's going to be *extremely* weird.

The approach presented here is the most in line with how the rest of the language works today.  Want an actual constant?   use 'const'.  Want something that can't be written to after a certain point?  use 'readonly'.